packages/relative-values/src/components/View/ListView/sidebar.tsx

```diff
@@ -0,0 +1,113 @@
+import { deflate } from "pako";
+import { ModelEvaluator } from "@/values/ModelEvaluator";
+import { Item } from "@/types";
+import { fromByteArray } from "base64-js";
+import { getModelCode } from "@/model/utils";
+import { FC, Fragment } from "react";
+import { NumberShower } from "@quri/squiggle-components";
+import { SquiggleChart } from "@quri/squiggle-components";
+
+function setHashData(str: string) {
```

Comment:
Maybe `getPlaygroundUrl`? `setHashData` makes me think that the function is not pure, but that's not the case; and "hash" is just an implementation detail.

Reply:
good point